src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Time.groovy

```diff
@@ -0,0 +1,13 @@
+package life.qbic.datamodel.dtos.imaging.properties
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
```

Comment:
Maybe not needed anymore, see my previous comment

Reply:
right, I removed it 👍